### ISSUE TITLE
Bitbucket Support

### ISSUE THREAD

**Issue by adamf321** [GoogleCloudPlatform/fourkeys#120]
Hi all,
Are there any plans to add bitbucket support to this project? I may be able to help with this too, if it's not too much work.

Thanks

**Comment by dinagraves** [comment]
Yes!  It's in the [roadmap](https://github.com/GoogleCloudPlatform/fourkeys/blob/main/ROADMAP.md#roadmap).  We would very much appreciate any help or contributions!

**Comment by john-landgrave** [comment]
Is there an interface (or set of interfaces) that define what adding a VCS system or a Bug Tracker looks like? Particularly since it looks like the `new_source.py` is [still a TODO](https://github.dev/GoogleCloudPlatform/fourkeys/blob/4d6241febe6fb012a488318f380816b8172106f8/README.md#L144) (#195), it looks like it would be:

- Add the new event source to the `sources.py` file [here](https://github.dev/GoogleCloudPlatform/fourkeys/blob/0ade327baf04036888cd9ccc801f5fce6bb579a2/event_handler/sources.py)
- Copying the `new-source-template` directory [here](https://github.com/GoogleCloudPlatform/fourkeys/tree/main/bq-workers) and then filling out the `process_new_source_event` that is presently a TODO
- Add the appropriate instructions into the README in `setup` [here](https://github.dev/GoogleCloudPlatform/fourkeys/blob/a896986f95ad2daa630452c42b8df3d7ba7188a8/setup/README.md)
- Add a `deploys_SERVICE` into the `deployments.sql` file [here](https://github.dev/GoogleCloudPlatform/fourkeys/blob/5e6e8e1330e1d931d8b21cfa7e128f1dcaeeeb39/queries/deployments.sql)
- Add a `WHEN` to each item in the SELECT clause that corresponds to the new event source in the `incidents.sql` file [here](https://github.dev/GoogleCloudPlatform/fourkeys/blob/6ce27fbb8ab50f9f8c8521354c6a620a06bfcc67/queries/incidents.sql)
- Update the `setup.sh` [here](https://github.dev/GoogleCloudPlatform/fourkeys/blob/20ba1992c98787b0859551a862b52429a6818925/setup/setup.sh)
- Add a `fourkeys-SERVICE-parser` module in the `terraform/modules` folder based on, for example, the `terraform/modules/fourkeys-github-parser` folder.


Is that about it? I would be interested in either adding Bitbucket and Jira myself or having someone in my organization do it, but I want to make sure that I have a reasonable size on the effort?
